player/ops/d_collecting.go

```diff
@@ -1,4 +1,62 @@
 package ops
 
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/adamhicks/ctrlaltdefeat/player"
+	"github.com/adamhicks/ctrlaltdefeat/player/config"
+	partsdb "github.com/adamhicks/ctrlaltdefeat/player/db/parts"
+	"github.com/adamhicks/ctrlaltdefeat/player/db/rounds"
+	"github.com/corverroos/unsure"
+	"github.com/corverroos/unsure/engine"
+)
+
 //CollectRoundsForever
 //Get PRs with state RoundCollecting, get the round parts, try and fetch missing data, or if complete transition to RoundCollected
+
+func CollectRoundsForever(c config.Config, backends Backends) error {
+	dbc := *backends.DB()
+	ctx := unsure.FatedContext()
+	ec := backends.EngineClient()
+	me := c.GetMe().Name
+	rs, err := rounds.ListWithStatus(ctx, dbc, player.PlayerRoundStatusRoundCollecting)
```

Comment:
this is a mistake in ListWithStatus, you shouldn't have to dereference .DB() so change the signature to accept a pointer